Makefile

```diff
@@ -44,6 +45,7 @@ clean:
 	make down
 	rm -rf $(PROJECT_ROOT)/webapp/src/shared/generated || true
 	rm $(PROJECT_ROOT)/transcendence-app/swagger-spec/swagger-spec.yaml || true
+	rm -rf $(PROJECT_ROOT)/packages/pong-engine/lib || true
```

Comment:
I think this is not needed because when we run npm run build-packages we are already deleting the generated code

Reply:
Done :+1: